crates/ethereum-rpc/src/gas_price/cache.rs

```diff
@@ -21,6 +21,7 @@ impl<C: sov_modules_api::Context> BlockCache<C> {
     }
 
     /// Gets block from cache or from provider
+    #[allow(unused)]
```

Comment:
Why is that?

Reply:
It is not used, but I think it is too early to remove unused parts related to gas price oracle. We should keep them temporarily until we see that current suggestion mechanism works fine.